tools/pylint-extensions/azure-pylint-guidelines-checker/README.md

```diff
@@ -92,3 +92,4 @@ In the case of a false positive, use the disable command to remove the pylint er
 | docstring-keyword-should-match-keyword-only        | Docstring keyword arguments and keyword-only method arguments should match.                                                                                          | pylint:disable=docstring-keyword-should-match-keyword-only          | [link](https://azure.github.io/azure-sdk/python_documentation.html#docstrings)                          |
 | docstring-type-do-not-use-class                    | Docstring type is formatted incorrectly. Do not use `:class` in docstring type.                                                                                      | pylint:disable=docstring-type-do-not-use-class                      | [link](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html)                            |
 | no-typing-import-in-type-check                     | Do not import typing under TYPE_CHECKING.                                                                                                                            | pylint:disable=no-typing-import-in-type-check                       | No Link.                                                                                                |
+| do-not-import-asyncio-directly                     | Do not import asyncio directly.                                                                                                                                      | pylint:disable=do-not-import-asyncio-directly                       | No Link.                                                                                                |
```

Comment:
I would probably just name the rule `do-not-import-asyncio`, and then leave the term "directly" in all the descriptions/comments.

Reply:
It has been renamed :-)